screensaver.turnoff/screensaver.py

```diff
@@ -1,164 +1,300 @@
+# -*- coding: utf-8 -*-
```

Comment:
Normally recommended to keep the entrypoint as small as possible. Keep it in mind for a future update.

Reply:
Ok.

Reply:
This will be fixed in the next release.
https://github.com/dagwieers/screensaver.turnoff/pull/13